# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/avifile/ff_eval_dup.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ffmpeg/libavutil/eval.c.orig	2005-12-07 16:09:07.000000000 +0100
+++ ./ffmpeg/libavutil/eval.c	2005-12-07 16:11:59.000000000 +0100
@@ -182,6 +182,8 @@
     return ret;
 }
 
+#if 0
+// this is already defined in ffmpeg/libavcodec/eval.c
 double ff_eval(char *s, double *const_value, const char **const_name,
                double (**func1)(void *, double), const char **func1_name,
                double (**func2)(void *, double, double), char **func2_name,
@@ -200,6 +202,7 @@
     
     return evalExpression(&p);
 }
+#endif
 
 #ifdef TEST
 #undef printf 
